resourcemgr/resourcemgr.c

```diff
@@ -245,7 +246,7 @@ typedef struct {
                                     //  flushed.
     UINT64 connectionId;            // Used to identify which connection owns the object,
                                     // sequence, or session.
-    void *nextEntry; // Next entry in the list; 0 to terminate list.                                
+    RESOURCE_MANAGER_ENTRY_PTR nextEntry; // Next entry in the list; 0 to terminate list.                             
 } RESOURCE_MANAGER_ENTRY;
```

Comment:
From looking just at the structure and it's name it's not entirely clear what a RESOURCE_MANAGER_ENTRY is (to me at least). The structure holds a pile of data and it all seems related to various handles that the RM may be switching in / out so an 'entry' would be a handle then?